load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -457,6 +482,30 @@ func (s *relaysSuite) mapStakingInfoWhenStakingAndDelegatingNewApps(
 func (s *relaysSuite) sendFundAvailableActorsTx(
 	plans *actorLoadTestIncrementPlans,
 ) (suppliers, gateways, applications []*accountInfo) {
+	// Send all the funding account's pending messages in a single transaction.
+	// This is done to avoid sending multiple transactions to fund the initial actors.
+	// pendingMsgs is reset after the transaction is sent.
+	defer s.sendPendingMsgsTx(s.fundingAccountInfo)
+	// Fund accounts for **initial** applications only.
+	// Additional applications are generated and funded as they're incremented.
+	for i := int64(0); i < s.appInitialCount; i++ {
+		// Determine the application funding amount based on the remaining test duration.
+		// for the initial applications, the funding is done at the start of the test,
+		// so the current block height is used.
+		appFundingAmount := s.getAppFundingAmount(s.testStartHeight)
+		// The application is created with the keyName formatted as "app-%d",
+		// starting from 1.
+		application := s.createApplicationAccount(i+1, appFundingAmount)
+		// Add a bank.MsgSend message to fund the application.
+		s.addPendingFundMsg(application.accAddress, sdk.NewCoins(application.amountToStake))
+
+		applications = append(applications, application)
+	}
+
+	if s.persistentChain {
```

Comment:
Anywhere we return early on `persistentChain` (or potentially epehmeralChain), we need to add a comment why.